src/index.js

```diff
@@ -1,3 +1,4 @@
+import './setupTests';
```

Comment:
why did you import this?

Reply:
because one of my test was not passing due to a fail which was "matchMedia not present, legacy browsers require a polyfill", so i searched for a solution about this i found something and it is to add this pieace of code inside the polyFill file: 

window.matchMedia =
  window.matchMedia ||
  function () {
    return {
      matches: false,
      addListener: function () {},
      removeListener: function () {},
    };
  };

and we didn't have polyFill file inside our app so another alternative was that to add this piece of code inside the setupTests.js and import it inside the index.js so this matchMedia testing fail won't happen again.